ddtrace/ext/tags.go

```diff
@@ -20,7 +20,7 @@ const (
 
 	// TargetPort sets the target host port.
 	// Deprecated: Use NetworkDestinationPort instead.
-	TargetPort = "out.port"
```

Comment:
Hmm I don't think this are the kind of breaking changes we should do in v2, as these could break stuff in the users monitors, dashboards, etc. - unless we decide is ok because no one is using these.

(also, in case we decide to go forward with this change, this constant should be entirely removed instead of the string value being changed, as this could be really confusing)